tiled/authenticators.py

```diff
@@ -29,6 +29,9 @@ class DummyAuthenticator:
 
     mode = Mode.password
 
+    def __init__(self, confirmation_message=""):
```

Comment:
Is an empty string default compliant? As in we only need to allow a confirmation message, not necessarily guarantee one?

Reply:
Perfect.